src/main/java/io/cryostat/net/reports/ActiveRecordingReportCache.java

```diff
@@ -134,7 +135,7 @@ protected String getReport(RecordingDescriptor recordingDescriptor, String filte
                 saveFile =
                         reportGeneratorServiceProvider
                                 .get()
-                                .exec(recordingDescriptor, filter)
+                                .exec(recordingDescriptor, filter, HttpMimeType.HTML.mime())
```

Comment:
If this new final method parameter represents a `Content-Type` or `MIME` type, why not pass the actual `HttpMimeType.HTML` enum value here rather than its `String` representation? This would be more type-safe and easier to reason about.